src/Mobs/Villager.h

```diff
@@ -38,19 +39,34 @@ class cVillager:
 	virtual void KilledBy    (TakeDamageInfo & a_TDI) override;
 
 	// cVillager functions
+	/** Returns the villager hidden inventory (8 slots). */
+	cItemGrid & GetInventory(void) { return m_Inventory; }
+
 	/** return true if the given blocktype are: crops, potatoes or carrots. */
 	bool IsBlockFarmable(BLOCKTYPE a_BlockType);
 
 	// Farmer functions
-	/** Searches in a 11x7x11 area for crops. If it found some it will navigate to them. */
+	static const int FARMER_RANDOM_TICK_SPEED = 5;
+	static constexpr double FARMER_SPECIAL_ACTION_CHANCE = 0.01;
+	/** This makes a 31x3x31 area. */
+	static constexpr Vector3i FARMER_SCAN_CROPS_AREA_RELATIVE {15, 1, 15};
+
+	/** Searches in a 31x3x31 area for crops. If it found some it will navigate to them. */
```

Comment:
The comment and the area don't match -

Reply:
I didn't see this comment before changing the area back... I think it's clearer the way it is now, although I agree the name was bad before. Better a Distance than an Area.